examples/wifi-echo/server/esp32/main/include/CHIPDeviceManager.h

```diff
@@ -77,7 +75,7 @@ class DLL_EXPORT CHIPDeviceManagerCallbacks
      * @param size               size of the attribute
      * @param value              pointer to the new value
      */
-    virtual void PostAttributeChangeCallback(uint8_t endpoint, ChipZclClusterId clusterId, ChipZclAttributeId attributeId,
+    virtual void PostAttributeChangeCallback(uint8_t endpoint, EmberAfClusterId clusterId, EmberAfAttributeId attributeId,
```

Comment:
Should we rename Ember to CHIP or have a Github issue to do so?

Reply:
I think it would be best to rename things in a different PR, this way there is a record that would help correlate the code to its Silabs' origin.

Reply:
The latter.  https://github.com/project-chip/connectedhomeip/issues/1610 filed.